### PR TITLE
lsusb for OSX

### PR DESCRIPTION
**Pull request by varunpatro**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/os-specific/darwin/lsusb/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    mkdir -p $out/share/man/man8
+    install -m 0755 lsusb $out/bin
+    install -m 0444 man/lsusb.8 $out/share/man/man8
+  '';
+
+  meta = {
+    homepage = https://github.com/jlhonora/lsusb;
+    description = "lsusb command for Mac OS X";
+    platforms = stdenv.lib.platforms.darwin;
+    license = stdenv.lib.licenses.mit;
```
Comment (matthewbauer) [inline pkgs/os-specific/darwin/lsusb/default.nix]: Can you add yourself as a maintainer here? Basically you'd just be responsible for version updates and making sure it builds.

**Review by matthewbauer — APPROVED** [review]

**Comment by varunpatro** [comment]
@matthewbauer what's the merging policy in `nixpkgs`? (how long on avg for PRs to get merged?) It's my first time doing this.

**Comment by LnL7** [comment]
@GrahamcOfBorg build darwin.lsusb

**Review by LnL7 — APPROVED** [review]

**Comment by LnL7** [comment]
Thanks for contributing!

I think trivial changes usually get merged in a few days. But it also depends a lot on how many committers are familiar with the particular package, etc. The number of people actively merging darwin related changes is pretty small for example.

**Pull request merged**